TestAssets/DesktopTestProjects/DesktopAppWithNativeDep/project.json

```diff
@@ -2,7 +2,7 @@
   "version": "1.0.0-*",
   "dependencies": {
     "PackageWithFakeNativeDep": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1"
+    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24611-07"
```

Comment:
The file is not correct. Until yesterday, CoreSetup 1.0.2 was supposed to contain the contents of corresponding 1.0.0 branches of CoreFX and CoreCLR.

However, the POR was updated to ensure that .NET Core 1.0.2 (from Core-Setup 1.0.0 branch) should only contain the fix for OSX PATH issue for Sierra and nothing else. Everything else will come in a subsequent LTS release (say 1.0.3).

Thus, Core-Setup 1.0.0 had bunch of commits reverted to no longer pickup the updated CoreCLR and CoreFX 1.0.0 packages, which is what the list above points to.

For CLI Preview 2, the change should simply be to refer to 1.0.2 M.N.App and that is it.